runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java

```diff
@@ -82,31 +82,34 @@ public Dimension render(Graphics2D graphics, Point parent)
 					: new net.runelite.api.Point(0, 0);
 
 				position.setLocation(mouseCanvasPosition.getX(), mouseCanvasPosition.getY());
+
+				// check if this tooltip would overlap the last
```

Comment:
As you are not using tooltips in the PR anymore, revert this commit as it is out of scope of the PR